### DISCUSSION TITLE
lively wallpaper Python & Node Js

### DISCUSSION THREAD

**Discussion by Eudritch** [rocksdanister/lively#751] [Q&A]
How can I use Python or Node Js to create a lively wallpaper? I went over the source code and couldn't figure out a single thing.

**Comment by rocksdanister** [comment]
https://github.com/rocksdanister/lively/wiki/Web-Guide-I-:-Cef

https://github.com/rocksdanister/lively/wiki/Application-Wallpaper

**Reply by Eudritch** [reply]
Thank-you that was quick

**Comment by erwinwallpaper** [comment]
@rocksdanister Thank You.
